aderyn_core/src/detect/low/_template.rs

```diff
@@ -22,6 +22,7 @@ pub struct TemplateDetector {
     // Keys are: [0] source file name, [1] line number, [2] character location of node.
     // Do not add items manually, use `capture!` to add nodes to this BTreeMap.
     found_instances: BTreeMap<(String, usize, String), NodeID>,
```

Comment:
I don't _love_ doubling up everywhere. Seems prone to mistakes / missing things.

Can we consolidate these into one?

Reply:
I agree. Let me try maintaining a hints dictionary in the detector structs (key -> error source location , value -> hints)

Reply:
This has not been resolved @TilakMaddy ?

Reply:
Sorry for leaving it hanging ... Moved to draft

Reply:
✅